Assets/_Project/fonts/JumpandPlay_Variant2.asset

```diff
@@ -73,7 +73,7 @@ Material:
     - _Reflectivity: 10
     - _ScaleRatioA: 0.9
     - _ScaleRatioB: 0.73125
-    - _ScaleRatioC: 0.33238634
```

Comment:
@Draekdude  Be very careful about committing font changes.

Unity keeps breaking the font files for no apparent reason so lets ignore these.

Reply:
![image](https://github.com/LPGameDevs/EditarrrPublic/assets/1744957/0e2a5508-22b9-421f-bf60-698022cb489d)

Reply:
OK, will do. That's odd.